casadm/cas_lib_utils.c

```diff
@@ -436,6 +436,22 @@ void *print_command_progress(void *th_arg)
  */
 int run_ioctl_interruptible(int fd, int command, void *cmd,
 		char *friendly_name, int cache_id, int core_id)
+{
+	bool retry = true;
+
+	return run_ioctl_interruptible_retry_option(int fd, int command, void *cmd,
+		char *friendly_name, int cache_id, int core_id, bool retry);
+}
+
+/*
+ * Run ioctl in a way that displays progressbar (if flushing operation takes longer)
+ * Catch SIGINT signal.
+ * @param friendly_name name of management operation that shall
+ * be displayed in command prompt
+ * @param retry decide if ioctl attepmts should retry
+ */
+int run_ioctl_interruptible_retry_option(int fd, int command, void *cmd,
```

Comment:
I'd make it separate functions with no additional arg.
Those running ioclt with default retry:
```
run_ioclt_retry()
run_ioclt_interruptible_retry()
```
and those running ioctl with no retry:
```
run_ioclt()
run_ioclt_interruptible()
```
That would be much easier to read.